documentation/docs/verifiable_credentials/create.mdx

```diff
@@ -12,15 +12,59 @@ keywords:
 import create_vc_rs from  '!!raw-loader!./../../../examples/account/create_vc.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
 
+You can create a Verifiable Credential with the following properties:
 
-TODO: Explain how a VC is created and verified.
+- [**Context**](https://www.w3.org/TR/vc-data-model/#contexts): list of JSON-LD context URIs. Includes `"https://www.w3.org/2018/credentials/v1"` by default.
```

Comment:
Do we have the API to actually adjust all of these fields? If we do, it is good that we mention them all like this, otherwise lets only list what can be changed.

Perhaps we should split it into a two categories. Right now it feels like an overwhelming list where some unimportant and some crucial fields are grouped. 

I'd suggest that we list important fields with a more detailed explaination:
* Subject
* Issuer
* Id
* Types

Next list other fields:
* Content
* All optionals

Reply:
Perhaps simply separate them by required and optional fields?

Reply:
> Do we have the API to actually adjust all of these fields?

I believe we do.

Reply:
#857